keck_vnc_launcher.py

```diff
@@ -1114,23 +1229,38 @@ def get_vnc_server(self, account, instrument):
         be out of date. As of this writing (July 2, 2020), ESI is incorrect on
         those machines, but other instruments return a correct server.
         '''
-        self.log.info(f"Determining VNC server for '{account}'...")
-        vncserver = None
-        for server in self.servers_to_try:
-            cmd = f'kvncinfo -server -I {instrument}'
 
-            try:
-                data = self.do_ssh_cmd(cmd, server, account)
-            except Exception as e:
-                self.log.error('  Failed: ' + str(e))
-                trace = traceback.format_exc()
-                self.log.debug(trace)
-                data = None
+        #cmd line option
+        if self.args.vncserver is not None:
+            self.log.info("Using VNC server defined on command line")
+            vncserver = self.args.vncserver
 
-            if data is not None and ' ' not in data:
-                vncserver = data
-                self.log.info(f"Got VNC server: '{vncserver}'")
-                break
+        #API Route
+        elif self.api_data:
+            self.log.info(f"Determining VNC server for '{self.args.account}' (via API)")
```

Comment:
Need to log what server we got, similar to line 1262 below.

Reply:
ok done